packages/parse-props/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@theme-ui/parse-props",
```

Comment:
I created a new package for the `parseProps()` functionality from `@theme-ui/core` which Theme UI puts in front of Emotion's jsx function. The reason for that is that we need a separate entry point for the new jsx transform, it needs to be a file or directory called `jsx-runtime` and `jsx-dev-runtime`. This is the only way I found to share the logic between the new and old jsx transform without duplicating the logic in the resulting bundles.

I'm not very happy about this approach though because this package is for internal use only and now it's exposed to the public. Maybe there is another way to solve this? Or maybe it's okay to ship the code twice?

Reply:
I wonder if we could use a *subpackage* approach similar to what emotion is doing. The only caveat might be `microbundle` which does not support multiple entry point.

Reply:
I think we could also just include the code in both bundles. `@theme-ui/parse-props` is roughly 300 Bytes now, shouldn't be too bad to have it in the bundle twice.